src/current/cockroachcloud/stream-changefeed-to-snowflake-aws.md

```diff
@@ -213,20 +199,46 @@ If your changefeed is running but data is not displaying in your S3 bucket, you
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > ALTER PIPE cdc_pipe refresh;
+    ALTER PIPE cdc_pipe refresh;
     ~~~
 
 1. To view the data in Snowflake, query the `order_alerts` table:
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > SELECT * FROM order_alerts;
+    SELECT * FROM order_alerts;
     ~~~
 
-    The ingested rows will display in the **Results** panel. It may take a few minutes for the data to load into the Snowflake cluster. To check on the progress of data being written to the destination table, see [Snowflake's documentation on querying a stage](https://docs.snowflake.com/user-guide/querying-stage.html).
+    The ingested rows will display in the **Results** panel. It may take a few minutes for the data to load into the Snowflake cluster. To check on the progress of data being written to the destination table, refer to [Snowflake's documentation on querying a stage](https://docs.snowflake.com/user-guide/querying-stage.html).
 
 Your changefeed is now streaming to Snowflake.
 
+## Remove multiple records from Snowflake target tables
+
+In some cases, you may need to remove multiple records for the same primary key caused by updates to a row. Although it is not possible to remove duplicates with Snowpipe before they arrive at the target table, Snowflake offers some features that you can use to automate removing multiple records from the table downstream.
+
+The following points outline two potential workarounds. For detailed instructions on each feature, we recommend reading the linked Snowflake documentation:
+
+- Use Snowflake [streams](https://docs.snowflake.com/en/user-guide/streams-intro) and [tasks](https://docs.snowflake.com/en/user-guide/tasks-intro) to write into a new de-duplicated table.
+    - Set up a stream that will watch for changes on the table that Snowpipe uploads into. The stream will create a snapshot of the changes at a specific point in time.
+    - Create the new table in Snowflake that will hold the de-duplicated entries using the stream's `METADATA$ACTION` column.
```

Comment:
```suggestion
    - Create a new table in Snowflake that will hold the de-duplicated entries using the stream's `METADATA$ACTION` column.
```

Reply:
"a" is fine here I think. Since we're creating "a" new table not referring back to "the" table we have already created.